apps/builddao/widget/components/modals/CreateProject.jsx

```diff
@@ -0,0 +1,486 @@
+  invalidAddress
+    ? setInvalidContributorFound(true)
+    : setInvalidContributorFound(false);
+  setContributors(filtered);
+};
+
+function isValidUrl(url) {
+  const regex = /^(?:(http|https):\/\/)?([^\s]+\.[^\s]+)?(?:\/[\w\-\.]+)*\/?$/;
+  return regex.test(url);
+}
+
+const websiteUrlHandler = (e) => {
+  const url = e.target.value;
+  setWebsite(url);
+};
+
+const projectAccountIdHandler = (e) => {
+  const accountId = e.target.value;
+  const isValid = isNearAddress(accountId);
+  !isValid ? setInvalidProjectAccount(true) : setInvalidProjectAccount(false);
+  setProjectAccount(accountId);
+};
+
+const Main = styled.div`
+  display: flex;
+  gap: 1.5rem;
+  padding: 1rem;
+
+  .form-control {
+    background: transparent;
+  }
+  .lhs {
+    width: 400px;
+    > div {
+      width: 100%;
+    }
+  }
+  .rhs {
+    width: 400px;
+  }
+
+  @media only screen and (max-width: 768px) {
+    flex-direction: column;
+    .lhs,
+    .rhs {
+      width: auto;
+    }
+  }
+
+  .form-group {
+    width: 100%;
+    & > div > div.p-2 {
+      background: transparent !important;
+      border: 1px solid var(--border-color, #ccc);
+      border-width: 0px 1px 1px;
+      border-top-style: initial;
+      border-right-style: solid;
+      border-bottom-style: solid;
+      border-left-style: solid;
+      border-top-color: initial;
+      border-right-color: rgb(222, 226, 230);
+      border-bottom-color: rgb(222, 226, 230);
+      border-left-color: rgb(222, 226, 230);
+      border-image: initial;
+      border-bottom-left-radius: 0.375rem;
+      border-bottom-right-radius: 0.375rem;
+      min-height: 9em;
+      * {
+        color: var(--text-color, #fff);
+      }
+    }
+    .nav-link {
+      color: var(--text-color, #fff);
+    }
+  }
+  .form-select {
+    background-color: #000 !important;
+    height: 100%;
+  }
+  .form-check {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding-left: 0px;
+  }
+  .form-check-input {
+    background-color: transparent;
+    border: 1px solid var(--border-color, #ccc);
+    &:checked {
+      background-color: var(--primary-color, #38c793);
+      border-color: var(--primary-color, #38c793);
+      &:focus {
+        box-shadow: none;
+      }
+    }
+    &:focus {
+      box-shadow: none;
+    }
+    &:not(:checked):not(:focus) {
+      background-image: url("data:image/svg+xml,%3csvg xmlns=%27http://www.w3.org/2000/svg%27 viewBox=%27-4 -4 8 8%27%3e%3ccircle r=%273%27 fill=%27%23fff%27/%3e%3c/svg%3e");
+    }
+  }
+  .location-team {
+    display: flex;
+    gap: 10px;
+    flex-direction: row;
+    @media only screen and (max-width: 768px) {
+      column-gap: 20px;
+      flex-direction: column;
+    }
+  }
+  .err,
+  .err-p_id {
+    color: #ff8888;
+    font-size: 12px;
+    padding: 0;
+    margin: 0;
+  }
+  .err-p_id {
+    margin-top: -20px;
+  }
+`;
+
+function onCreateProject() {
+  const projectID = normalize(title);
```

Comment:
![Screenshot 2024-04-22 at 9 23 45 AM](https://github.com/NEARBuilders/gateway/assets/16282460/2aaac501-de50-4f22-b614-6759d87e5dad)

When saving a project: 

projectId is undefined, also the key builddao.testnet_project_ is missing

Reply:
I think this is done too, looking at your last commit